README.md

```diff
@@ -262,7 +262,7 @@ Then we need to send a `cursor-position` event to the session backend as our cur
 We can do this by importing the `useSend` hook and then creating a `sendEvent` function with it:
 
 ```ts filename="src/components/Home.tsx" {1, 5}
-import { SessionBackendProvider, useEventListener, useReady, useSend } from '@jamsocket/javascript/react'
+import { useEventListener, useSend } from '@jamsocket/javascript/socketio'
```

Comment:
I think we should include the `import { SessionBackendProvider } from '@jamsocket/javascript/react'` right above this, too. Note: we'll need to update the highlighted line numbers.